### ISSUE TITLE
[Suggestion] Could the GE plug-in be changed so it's more clear? (with picture example) 

### ISSUE THREAD

**Issue by yaze1** [runelite/runelite#3207]
https://imgur.com/a/9Ylk5vx

**Letters** should be **bigger** (specifically the amount of the item and wether it's buying or selling).
The **progress bar** should be **bigger**, (when there's progress, two clearly distinguishable colors) and thus more _visible_.

I type should, but really I mean, pretty please?

peace

**Comment by Hydrox6** [comment]
Do not make duplicate issues, please

**Comment by deathbeam** [comment]
Currently it perfectly fits sidebar in fixed mode, so, no. You already created similar issue dont you.

**Comment by yaze1** [comment]
No no I removed what shouldn't be changed in previous post. Look two different pictures. I cut out the non essential and left the essential. not duplicate

**Comment by Hydrox6** [comment]
This is a duplicate. Same title, same request.

**Comment by yaze1** [comment]
Deathbeam, read the post, does it say anything about fitting the sidebar in fixed mode? No, that was in the previous post, which I did not copy paste entirely, I cut exactly what you are talking about here.

**Comment by deathbeam** [comment]
As i said, currently the interface fits perfectly. With your proposals this design will break most likely. Also, we discussed the look of the GE interface already before update happened, I dont think this will be implemented. But, I guess I can reopen, why not

**Comment by yaze1** [comment]
Nope, please read both posts and pictures and see what is changed: I no longer ask for size to be to fit sidebar in fixed mode. I ask for progress bar to be made bigger and different colors so it's more visible how much progress you got. Also, I ask for the amount and wether it's buying or selling to be made bigger so that's also more clearly visible. Check picture to see how big.

**Comment by yaze1** [comment]
I don't know what is possible with code, I can only ask..
